docs/user/troubleshooting.md

```diff
@@ -89,3 +89,45 @@ This is safe to ignore and merely indicates that the etcd bootstrapping is still
 ### Installer Fails to Create Resources
 
 The easiest way to get more debugging information from the installer is to increase the logging level. This can be done by adding `--log-level=debug` to the command line arguments. Of course, this cannot be retroactively applied, so it won't help to debug an installation that has already failed. The installation will have to be attempted again.
+
+## Generic Troubleshooting
+
+Here are some ideas if none of the [common failures](#common-failures) match your symptoms.
+
+### Check for Pending or Crashing Pods
+
+This is the generic version of the [*No Worker Nodes Created*](#no-worker-nodes-created) troubleshooting procedure.
+
+```console
+$ oc --config=${INSTALL_DIR}/auth/kubeconfig get pods --all-namespaces
+NAMESPACE                              NAME                                                              READY     STATUS              RESTARTS   AGE
+kube-system                            etcd-member-wking-master-0                                        1/1       Running             0          46s
+openshift-cluster-api                  machine-api-operator-586bd5b6b9-bxq9s                             0/1       Pending             0          1m
+openshift-cluster-dns-operator         cluster-dns-operator-7f4f6866b9-kzth5                             0/1       Pending             0          2m
+...
+```
+
+You can investigate any pods listed as `Pending` with:
+
+```sh
+oc --config=${INSTALL_DIR}/auth/kubeconfig describe -n openshift-cluster-api pod/machine-api-operator-586bd5b6b9-bxq9s
+```
+
+which may show events with warnings like:
+
+```
+Warning  FailedScheduling  1m (x10 over 1m)  default-scheduler  0/1 nodes are available: 1 node(s) had taints that the pod didn't tolerate.
+```
+
+### Check for Unready Nodes
```

Comment:
the describe on node will give you conditions for the node object that will provide information as to why the node is not ready. it might be network not ready, disk pressure etc.. again i think we should point to the docs on kubelet node conditions.

Reply:
https://kubernetes.io/docs/concepts/architecture/nodes/#node-status

Reply:
> the describe on node will give you conditions...

Obsolete now that I've [dropped this section][1].

[1]: https://github.com/openshift/installer/pull/664#issuecomment-438425041